contracts/contracts/trees/AccQueue.sol

```diff
@@ -442,31 +433,25 @@ abstract contract AccQueue is Ownable, Hasher {
     }
   }
 
-  /*
-   * Returns the subroot at the specified index. Reverts if the index refers
-   * to a subtree which has not been filled yet.
-   * @param _index The subroot index.
-   */
+  /// @notice Returns the subroot at the specified index. Reverts if the index refers
+  /// to a subtree which has not been filled yet.
+  /// @param _index The subroot index.
   function getSubRoot(uint256 _index) public view returns (uint256) {
     if (currentSubtreeIndex <= _index) revert InvalidIndex(_index);
     return subRoots[_index];
   }
 
-  /*
-   * Returns the subroot tree (SRT) root. Its value must first be computed
-   * using mergeSubRoots.
-   */
+  /// @notice Returns the subroot tree (SRT) root. Its value must first be computed
+  /// using mergeSubRoots.
   function getSmallSRTroot() public view returns (uint256) {
     if (!subTreesMerged) revert SubTreesNotMerged();
     return smallSRTroot;
   }
 
-  /*
-   * Return the merged Merkle root of all the leaves at a desired depth.
-   * merge() or merged(_depth) must be called first.
-   * @param _depth The depth of the main tree. It must first be computed
-   *               using mergeSubRoots() and merge().
-   */
+  /// @notice Return the merged Merkle root of all the leaves at a desired depth.
+  /// merge() or merged(_depth) must be called first.
```

Comment:
<img width="897" alt="image" src="https://github.com/privacy-scaling-explorations/maci/assets/9023842/24d060c6-3e10-4445-be26-d2dfbd9ce036">

maybe add an empty /// line to prevent the notice displayed in the same line?